runtime/compiler/control/J9Options.cpp

```diff
@@ -898,6 +933,12 @@ TR::OptionTable OMR::Options::_feOptions[] = {
         TR::Options::setStaticNumeric, (intptr_t)&TR::Options::_iprofilerSamplesBeforeTurningOff, 0, "P%d", NOT_IN_SUBSET},
    {"itFileNamePrefix=",  "L<filename>\tprefix for itrace filename",
         TR::Options::setStringForPrivateBase, offsetof(TR_JitPrivateConfig,itraceFileNamePrefix), 0, "P%s"},
+#if defined(J9VM_OPT_JITSERVER)
+   {"jitserverAOTCacheLoadExclude=", "D{regex}\tdo not load methods matching regex from the JITServer AOT cache",
```

Comment:
They are technically not regular expressions either, but since many other options refer to these patterns as "regex", it's fine if we do that as well.

Reply:
I think they at least support character sets as well, but yes, not much more than that.